persistence/mongodb/migrating-from-tekmaven.md

```diff
@@ -43,3 +33,39 @@ class MySagaData : IContainSagaData
 ### How Document Versioning Works
 
 include: document-version
+
+
+## Compatibility mode
+
+The compatibility mode enables the MongoDB persistence to work with existing saga data without modifications to the database.
+
+### Configuration
+
+Use the following compatibility API to configure the package to work with existing saga data:
+
+snippet: MongoDBTekmavenCompatibility
+
+The `VersionElementName` value must match the `BsonDocument` element name used by the previous saga data property decorated with the `[DocumentVersion]` attribute.
+
+include: must-apply-conventions-for-version
+
+
+## Migrating data
+
+Alternatively to the compatibility mode, the existing saga data can be migrated to the data format used by the `NServiceBus.Storage.MongoDB` package. This approach requires the endpoint to be stopped during the migration. Use the `mongo` Shell to connect to the database and execute the following script:
```

Comment:
Technically, they don't need to stop the endpoint though doing it in-flight introduces some challenges. I think we're okay to leave the recommendation as-is but mentioning it in case others disagree.

Reply:
yeah I think it's safer to recommend to stop and migrate rather than doing this on a live system. if they are experienced with MongoDB they would figure this out themselves I think?